src/components/Matrix/TableCell/index.js

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import classnames from "classnames";
+import styles from "./TableCell.module.scss";
+
+export const TableCell = ({
+  children,
+  data,
+  style,
+  onClick,
+  findClosestIds,
+  className,
+  onMouseEnter,
+  onMouseLeave,
+  onSetClosestCellIds,
+}) => {
+  const [isHover, setIsHover] = useState(false);
+
+  const handleEnter = () => {
+    if (findClosestIds) {
+      setIsHover(true);
+      findClosestIds();
+    } else if (onMouseEnter) {
+      onMouseEnter();
+    }
+  };
+
+  const handleLeave = () => {
+    if (onSetClosestCellIds) {
+      setIsHover(false);
+      onSetClosestCellIds([]);
```

Comment:
that logic should be inside onMouseLeave

Reply:
This can`t be inside onMouseLeave, because this function only uses when we hover on SumCell and contain only setData